### PR TITLE
[GCI 2019] [Task] Add Screenshots

### PR DESCRIPTION
**Pull request by trungnt2910**
I have added a screenshot, as well as a README.md file for DeskNotes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## DeskNotes
+DeskNotes is a tool to put simple sticky notes on the Desktop
```
Comment (humdingerb) [inline README.md]: Full stop at the end.

**Review by humdingerb — CHANGES_REQUESTED** [review]
It'd be nice to mention that you can change background and text colour. Then create 3 or 4 note windows slightly overlapping each other with different colours (like yellow, green, magenta). Arrange those windows on the single colour Desktop background, that makes it easy to use WonderBrush to make it transparent.
I can guide you further, if you need helpp with WonderBrush.

File: README.md (hunk 1 above)
Comment (trungnt2910) [reply README.md]: I have updated the screenshot to show this feature.

**Review by humdingerb — CHANGES_REQUESTED** [review]
Can you grab the whole windows of the notes? Just stack'em loosely and grab the whole screen, then edit in e.g. WonderBrush.

**Comment by trungnt2910** [comment]
> the whole windows of the notes? Just stack'em loosely and grab the whole screen, then edit in e.g. WonderBrush.

So you mean that I should take the screenshots of the windows instead of the Replicants?

**Comment by trungnt2910** [comment]
> Can you grab the whole windows of the notes? Just stack'em loosely and grab the whole screen, then edit in e.g. WonderBrush.

If that's what you mean, then it's not possible. DeskNotes only allows one running window, but many replicants. So should I take many screenshots of windows with different colors, then combine them with WonderBrush?

**Comment by humdingerb** [comment]
> DeskNotes only allows one running window, but many replicants. 

Ah, right. Maybe a little heap of replicants only, no window. You may even arrange them on the default blue background with large Home, Trash and Haiku icon in the shot, so we see that we're on the Desktop. Thanks!

**Comment by trungnt2910** [comment]
> DeskNotes only allows one running window, but many replicants.
> 
> Ah, right. Maybe a little heap of replicants only, no window. You may even arrange them on the default blue background with large Home, Trash and Haiku icon in the shot, so we see that we're on the Desktop. Thanks!

Yes I have put the heap of replicants on the Desktop in the latest screenshot.

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**